src/components/Loader/Loader.test.tsx

```diff
@@ -0,0 +1,8 @@
+import { render } from '@testing-library/react';
+import { expect, test } from 'vitest';
+import Loader from './Loader';
+
+test('renders a loader', () => {
+  const loader = render(<Loader />);
+  expect(loader).toBeDefined();
+});
```

Comment:
Этот тест практически бесполезен. expect(loader).toBeDefined() проверяет только то, что render() вернул объект, что всегда будет true. Тест не проверяет, что компонент реально отрендерился или работает корректно.

```suggestion
import '@testing-library/jest-dom/vitest';
import { render, screen } from '@testing-library/react';
import { expect, test } from 'vitest';
import Loader from './Loader';

// Вариант 1: Проверка текста
test('renders loader text', () => {
  render(<Loader />);
  expect(screen.getByText('Loading...')).toBeInTheDocument();
});

// Вариант 2: Проверка по role или test-id
test('renders loader element', () => {
  render(<Loader />);
  expect(screen.getByRole('status')).toBeInTheDocument();
  // или
  expect(screen.getByTestId('loader')).toBeInTheDocument();
});

// Вариант 3: Проверка CSS класса или стилей
test('renders loader with correct styling', () => {
  render(<Loader />);
  const loader = screen.getByRole('status');
  expect(loader).toHaveClass('loader');
});
```